README.md

```diff
@@ -11,6 +11,7 @@ Supported editors:
 - [PhpStorm](https://www.jetbrains.com/phpstorm/)
 - [IDEA 14 Community Edition](https://www.jetbrains.com/idea/download/)
 - [Vim](http://www.vim.org/)
+- [Emacs](https://www.gnu.org/software/emacs/)
```

Comment:
It also should be added to `editor` setting value list (https://github.com/lahmatiy/open-in-editor#editor)